exmg-snackbars-icons.ts

```diff
@@ -0,0 +1,16 @@
+import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
+const documentContainer = document.createElement('div');
+documentContainer.setAttribute('style', 'display: none;');
+
+documentContainer.innerHTML = `<iron-iconset-svg name="exmg-icons" size="24">
```

Comment:
Can you use icons without iron-icon-set

For example
import { html } from 'lit-element';
export const menuIcon = html`<svg height="24" viewBox="0 0 24 24" width="24"><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path></svg>`;

Reply:
done